build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: vpcnetworkconfigurations.nsx.vmware.com
+spec:
+  group: nsx.vmware.com
+  names:
+    kind: VPCNetworkConfiguration
+    listKind: VPCNetworkConfigurationList
+    plural: vpcnetworkconfigurations
+    singular: vpcnetworkconfiguration
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - description: Edge Cluster to be used
+      jsonPath: .spec.EdgeCluster
+      name: EdgeCluster
+      type: string
+    - description: NSXT Project the Namespace associated with
+      jsonPath: .spec.NSXTProject
+      name: NSXT Project
+      type: string
+    - description: PublicIPv4Block assigned to the Namespace
+      jsonPath: .spec.PublicIPv4Block
+      name: PublicIPv4Block
+      type: string
+    - description: PrivateCIDRs assigned to the Namespace
+      jsonPath: .spec.PrivateCIDRs
+      name: Private CIDRs
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: VPCNetworkConfiguration is the Schema for the vpcnetworkconfigurations
+          API.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: VPCNetworkConfigurationSpec defines the desired state of
+              VPCNetworkConfiguration. There is a default VPCNetworkConfiguration
+              that applies to Namespaces do not have a VPCNetworkConfiguration assigned.
+              When a field is not set in a Namespace's VPCNetworkConfiguration, the
+              Namespace will use the value in the default VPCNetworkConfiguration.
+            properties:
+              appliedToNamespaces:
+                description: Namespaces to be applied. The default VPCNetworkConfiguration
+                  will not have AppliedToNamespaces set.
+                items:
+                  type: string
+                type: array
+              defaultSubnetPrefixLength:
+                default: 28
```

Comment:
I have the same comment in Subnet CRD definition, do you think `defaultSubnetPrefixLength` should be the same with the default size of subnet?

Reply:
As we discussed, I think the defaultSubnetPrefixLength in this CRD is used for the auto-created SubnetSet CR in each namespace. default_subnet_size in Subnet CR is used when user creates Subnet CR without specifying the value.
So the default value can be different.

Reply:
changed default value to /26 for defaultSubnetPrefixLength to be consistent with the default subnet_size in Subnet CR.

Reply:
Will there be migration problem in future if we change default value from /28 to /26?

Reply:
I think it is OK. When migration in future,  we can set the defaultSubnetPrefixLength as 28 for the NSXNetworkConfiguration CR migrated.  Do you think it is OK?

Reply:
Sounds good to me, thanks